test/functional/Java8andUp/src/org/openj9/test/java/lang/Test_RuntimePermission.java

```diff
@@ -21,6 +21,13 @@
  *******************************************************************************/
 package org.openj9.test.java.lang;
 
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+import org.testng.annotations.BeforeMethod;
+import org.testng.AssertJUnit;
+
+package org.openj9.test.java.lang;
```

Comment:
@dhong44 some test failures because of this. The `package` is already defined. Looks like a copy/paste error. Can you make the appropriate fixes and I'll relaunch the builds.

Reply:
1a2c592